it/src/test/java/com/expediagroup/streamplatform/streamregistry/it/StreamRegistryIT.java

```diff
@@ -66,8 +68,13 @@ public static void before() {
         "--spring.jpa.show-sql=false"
     };
     context = SpringApplication.run(StreamRegistryApp.class, args);
-    final String url = String.format("http://localhost:%s%s", context.getEnvironment().getProperty("local.server.port"), "/graphql");
+
+    final String port = context.getEnvironment().getProperty("local.server.port");
+    final String endpoint = "/graphql";
+    final String url = String.format("http://localhost:%s%s", port, endpoint);
+
     client = new ITestClient(url);
+    TimeUnit.SECONDS.sleep(2);
```

Comment:
A NPE is thrown during travis execution, it is a constant issue in that section (I have seen it in other PR's than mine), that's why I broke url in several lines and I figured out that method is (in some way) taken out of the stack before `StreamRegistryApolloClient` component has been built, invalidating the `url` parameter after its constructor is called, otherwise NPE doesn't have sense because `url` is never null.
I decided to add a bit delay to give more chance `StreamRegistryApolloClient` of been build without receiving a null parameter. 

**log**:
```
[ERROR] com.expediagroup.streamplatform.streamregistry.it.StreamRegistryIT  Time elapsed: 25.417 s  <<< ERROR!
java.lang.NullPointerException: serverUrl is null
	at com.expediagroup.streamplatform.streamregistry.it.StreamRegistryIT.before(StreamRegistryIT.java:76)
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   StreamRegistryIT.before:76 » NullPointer serverUrl is null
```
**First constructor using `serverUrl` as variable**:
```
  public StreamRegistryClient(final String url) {
    this(StreamRegistryApolloClient.builder().serverUrl(url).build());
  }
```